deploy/olm-catalog/ocs-operator/manifests/ocs-operator.clusterserviceversion.yaml

```diff
@@ -1904,7 +1904,7 @@ spec:
                     fieldRef:
                       fieldPath: metadata.namespace
                 - name: CSI_ENABLE_SNAPSHOTTER
-                  value: "false"
+                  value: "true"
```

Comment:
@aruniiird Thanks for the PR, we have provided this option to disable snapshot as it was in alpha version and it's going to be the default. I thought to remove this flag in the next ROOK version as it is going to be backward compatible and enabled always in OCS.

@travisn Any suggestion?

 apart from this change, we need to create and delete snashotclass for both cephfs and rbd.

Reply:
@Madhu-1 Per discussion in the csi meeting, agreed that we can just remove this setting from rook and always have the snapshotter enabled. Nobody outside OCS was using that feature to disable it.